source/Meadow-Filter-Parse.js

```diff
@@ -160,6 +160,47 @@ const addFilterStanzaToQuery = (pFilterStanza, pQuery) =>
 			}
 			break;
 
```

Comment:
Missing documentation for new stanzas here: https://github.com/bigal-hl/meadow-filter/blob/main/source/Meadow-Filter-Parse.js#L45